README.md

```diff
@@ -43,12 +53,15 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 
-The rights over this page are reserved for [@Cindy_Shin] (https://www.behance.net/adagio07)
+👤 **Sergio Valadez**
 
-👤 **Author**
+- GitHub: [@githubhandle](https://github.com/sergiogval)
+- Twitter: [@twitterhandle](https://twitter.com/sgvldz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-- GitHub: [@sergiogval](https://github.com/sergiogval)
-- Instagram: [@sergioval](https://instagram.com/sergioval)
+👤 **Cindy Shin**
+
+- Original design idea
 
 ## 🤝 Contributing
```

Comment:
- [ ] One of the requirements says that The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Thanks for this review.